Viewer/src/net/relinc/viewer/application/ScaledResults.java

```diff
@@ -17,20 +17,18 @@
 	private double[] strainRate;
 	private double[] frontFaceForce;
 	private double[] backFaceForce;
-
+	private Sample sample;
 
 	public ScaledResults(Sample s) {
+		this.sample = s;
```

Comment:
This is a tiny bit risky.. The Sample object could change under our feet between calls to the `ScaledResults` constructor and `getFrontFaceForce`. But the risk seems pretty low. It seemed to work for me. So this optimization is not totally "free" in terms of keeping the behavior of the program the same